proxy/src/main/java/net/md_5/bungee/BungeeCord.java

```diff
@@ -586,18 +587,26 @@ public int getOnlineCount()
     }
 
     @Override
-    public ProxiedPlayer getPlayer(String name)
+    public Collection<ProxiedPlayer> getPlayers(String name)
     {
         connectionLock.readLock().lock();
         try
         {
-            return connections.get( name );
+            return connections.values().stream()
+                .filter( player -> player.getName().equals( name ) )
+                .collect( Collectors.toList() );
         } finally
         {
             connectionLock.readLock().unlock();
         }
     }
 
+    @Override
+    public ProxiedPlayer getPlayer(String name)
+    {
+        return getPlayers( name ).iterator().next();
```

Comment:
Iterator.next throws an exception on empty list - use hasNext on it before and if missing, return null